test/time_tracker/timers/pubsub_test.clj

```diff
@@ -1,6 +1,7 @@
 (ns time-tracker.timers.pubsub-test
   (:require [clojure.test :refer :all]
             [clojure.core.async :refer [chan alt!! put!] :as async]
+            [clojure.string :as str]
```

Comment:
@jysandy @trycatcher That's fine. Choose something else if you have spec in the same namespace. `string` is probably a bit verbose*, but it works. Just don't use `:refer-clojure :exclude`.

* The reason `string` is verbose is because you'll likely move all your string manipulation into a few namespaces, over time. These namespaces will be full of `string/x` `string/y` everywhere, which will make its awkwardness apparent when we cross that bridge. In those cases, you probably want to graduate to `refer :only`.

If you look up the ClojureDocs on `refer`, all the examples are from `clojure.string`, which seems telling. :) https://clojuredocs.org/clojure.core/refer